address.go

```diff
@@ -1,11 +1,28 @@
 package faker
 
 import (
+	_ "embed"
+	"encoding/json"
+	"github.com/go-faker/faker/v4/pkg/options"
 	"reflect"
```

Comment:
Let's separate import to be tidier
1. Go's package
2. External package
3. Internal package
```suggestion
	_ "embed"
	"encoding/json"
	"reflect"
	
	"github.com/go-faker/faker/v4/pkg/options"
```

Reply:
using gofmt solve this